docs/source/usage/how_to_run.rst

```diff
@@ -3,87 +3,89 @@
-#. add an **inputs file** and on :ref:`HPC systems <install-hpc>` a **submission script** to the directory
-#. run
+When to run: Physics Models
+---------------------------
 
-1. Run Directory
-----------------
+ImpactX most powerful and central physics model is its symplectic particle tracker with collective effects (space charge, CSR, wakefields, etc.).
+Additionally, ImpactX provides two simplified physics models to aid scientists through every step, from beamline inception to operation:
+only tracking of the reference particle orbit, or tracking of the beam envelope (covariance matrix) through linearized transport maps.
 
-On Linux/macOS, this is as easy as this
+================== =============== =============== ==================
+Mode               Use Case        Generality      Collective Effects
+================== =============== =============== ==================
+Particle Tracking  Full Dynamics   Most general    Supported
+Reference Tracking Early Design    Reference orbit No
+Envelope Tracking  Rapid Scans     Linearized      Not yet
+================== =============== =============== ==================
 
-.. code-block:: bash
 
-   mkdir -p <run_directory>
+What to run: User Interfaces
+----------------------------
 
-Where ``<run_directory>`` by the actual path to the run directory.
+ImpactX can be run in three major interfaces:
 
-2. Executable
--------------
+#. As a Python module (script),
+#. As an executable application with a key-value input file, or
+#. From a :ref:`graphical user interface (dashboard) <usage-dashboard>`.
 
-If you installed ImpactX with a :ref:`package manager <install-users>`, a ``impactx``-prefixed executable will be available as a regular system command to you.
-Depending on the choosen build options, the name is suffixed with more details.
-Try it like this:
+While each of these interfaces provide access to the same physics models, you might pick one or the other for specific needs, experience and workflows:
 
-.. code-block:: bash
+=========== ================= ========================= =========== =========== ============== ===========
+Interface   Experience        Good For                  Interactive Extensible  Jupyter        HPC Support
+=========== ================= ========================= =========== =========== ============== ===========
+Python      Beginner-Advanced Automation, AI/ML         Yes         Yes         Yes            Yes
+Application Advanced          Minimal requirements      No          No          Terminal       Yes
+Dashboard   Beginner          Learning, Control-Systems Yes         No          Yes            Not yet
+=========== ================= ========================= =========== =========== ============== ===========
 
-   impactx<TAB>
 
-Hitting the ``<TAB>`` key will suggest available ImpactX executables as found in your ``PATH`` `environment variable <https://en.wikipedia.org/wiki/PATH_(variable)>`__.
+How to run
```

Comment:
I propose moving the "How to run" section to the end of this page, and specifying that the steps described are given for a local computer.  For details on how to proceed on HPC machines, we can add a link to the existing documentation.

Reply:
Agreed. Let us quickly connect if you would do another change here to discuss.

Reply:
Resolved offline: ok for now, we will do a theory section update as a follow-up and will refine docs further.